app/assets/templates/single-post-viewer/single-post-interaction-counts_tpl.jst.hbs

```diff
@@ -0,0 +1,42 @@
+{{#if resharesCount}}
+  <div id="reshares" class="clearfix">
+    <div class="count">
+      <i class="entypo-reshare middle gray"></i>
+      <span>{{resharesCount}}</span>
+    </div>
+    <div class="interaction-avatars">
+      {{#each reshares}}
+        {{#linkToAuthor author}}
+          {{{personImage this 'small' 'micro'}}}
```

Comment:
Handlebar helpers support double qoutes. Might be a good place to start adopting if we're on that file.